.eslintrc.js

```diff
@@ -2,8 +2,19 @@
 
 const config = {
   root: true,
-  env: { commonjs: true, node: true, browser: true },
-  extends: ['ackama', 'ackama/react'],
+  env: {
+    commonjs: true,
+    node: true,
+    browser: true,
+    jest: true
+  },
+  extends: [
+    'ackama',
+    'ackama/react',
+    'plugin:testing-library/recommended',
+    'plugin:testing-library/react'
```

Comment:
We should use our `ackama/jest` config, which should be applied to only test files using `overrides` - this also applies to `testing-library` and co configs + plugins.

i.e

```
  ...
  extends: ['ackama', 'ackama/@typescript-eslint'],
  ignorePatterns: [],
  overrides: [
    {
      files: ['test/**'],
      extends: ['ackama/jest'],
      rules: { 'jest/prefer-expect-assertions': 'off' }
    }
  ],
  ...
```

Reply:
> Got a few things I'd like addressed, but overall looking good!
> 
> I'm neutral on where you've actually put the tests - in javascript projects we tend to put them in `test/`, and I think having them at the root would make them more discoverable, _but_ this is a whole new thing for us and it'd potentially have to share that space with other testing tools, so on the whole I'm happy with seeing how this goes.
> 
> Do you want to have a crack at the typescript version in a followup PR? otherwise I can do it :)
> (I'll merge the TS variant now)

I can see about adding the TS variant tomorrow morning if I get some time

Reply:
No rush - it'll probably be worth landing in its own PR to keep the noise down anyway, once this one is merged.